### PR TITLE
Fix a wrong object key

### PR DESCRIPTION
**Pull request by zamotkin**
A wrong object key when using transform object keys and unicode escape sequence options at once #494

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -447,11 +447,6 @@
   resolved "https://registry.yarnpkg.com/@xtuc/long/-/long-4.2.2.tgz#d291c6a4e97989b5c61d9acf396ae4fe133a718d"
```
Comment (sanex3339) [inline yarn.lock]: I think this file should not be updated

Comment (zamotkin) [reply yarn.lock]: Yes, it's a very strange. I've tried to drop node_modules folder in master and re-run yarn. I still have the same result, yarn.lock changed. Can you test on your side?

Comment (sanex3339) [reply yarn.lock]: What yarn version are you using?

Comment (zamotkin) [reply yarn.lock]: 1.19.2

Comment (zamotkin) [reply yarn.lock]: The same behaviour with the latest 1.21.1

Comment (sanex3339) [reply yarn.lock]: The same for me with `rm -rf ./node_modules && yarn`

File: test/functional-tests/node-transformers/converting-transformers/object-expression-transformer/ObjectExpressionTransformer.spec.ts [hunk 2]
```diff
@@ -199,4 +199,37 @@ describe('ObjectExpressionTransformer', () => {
             assert.match(obfuscatedCode, objectSpreadRegExp);
         });
     });
+
+    describe('object spread: unicode escape sequence', () => {
```
Comment (sanex3339) [inline test/functional-tests/node-transformers/converting-transformers/object-expression-transformer/ObjectExpressionTransformer.spec.ts]: this test is not covered bug. If following line will removed
```
this.stringsCache.set(`${result}-${String(encodeAllSymbols)}`, result);
```
test still will pass

**Pull request merged**